File: Arrays - Distinct Elements Count.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int l1 = sc.nextInt(), l2 = sc.nextInt();
        Set<Integer> s1 = new HashSet<>(), s2 = new HashSet<>();
        
        for (int i = 0; i < l1; i++) s1.add(sc.nextInt());
        for (int i = 0; i < l2; i++) s2.add(sc.nextInt());
        
        Set<Integer> d1 = new HashSet<>(s1), d2 = new HashSet<>(s2);
        d1.removeAll(s2);
        d2.removeAll(s1);
        
        System.out.println(d1.size() + d2.size());
    }
}

Example Input/Output 1:
Input:
4 5
1 5 9 10
22 5 12 95
Output:
4
Example Input/Output 2:
Input
3 2
100 200 300
201 600
Output:
5

File: Arrays - Distinct Elements Count.cpp
#include <iostream>
#include <unordered_set>
using namespace std;

int main() {
    int a, b;
    cin >> a >> b;

    unordered_set<int> x, y;

    for (int i = 0, z; i < a; i++) {
        cin >> z;
        x.insert(z);
    }
    for (int i = 0, z; i < b; i++) {
        cin >> z;
        y.insert(z);
    }

    unordered_set<int> p = x, q = y;
    for (int z : y) p.erase(z);
    for (int z : x) q.erase(z);

    cout << p.size() + q.size() << endl;
    return 0;
}

Example Input/Output 1:
Input:
4 5
1 5 9 10
22 5 12 95
Output:
4
Example Input/Output 2:
Input
3 2
100 200 300
201 600
Output:
5
